### DISCUSSION TITLE
n8n.io integration

### DISCUSSION THREAD

**Discussion by iodize6399** [Budibase/budibase#2566] [Ideas]
This is a continuation of the [now locked suggestion](https://github.com/Budibase/budibase/discussions/891#discussioncomment-381459) thread about the n8n.io integration. I couldn't find it mentioned here in Ideas, so made a feature request just to highlight and document it further.

**Comment by mjashanks** [comment]
It's been on our list for a long time! 

Thanks for re-raising, we do want to get this done.

**Comment by tluyben** [comment]
What would integration look like outside what already is possible? I find the n8n pretty horrible but it is not about me; our lawyers simply say 'nope, way too vague' but it depends on the level of integration of course.

**Reply by mjashanks** [reply]
Hey @tluyben - apologies, but I don't know what you're asking here :)

At the moment, we don't have an n8n integration, although the REST connector might cover it.
